orga/tests/integration/components/routes/authenticated/campaigns/new-item-test.js

```diff
@@ -76,6 +76,21 @@ module('Integration | Component | routes/authenticated/campaign | new-item', fun
       assert.dom('#campaign-title').exists();
       assert.dom('#campaign-target-profile').exists();
     });
+
+    test('it should display documentation of school paths', async function(assert) {
+      // given
+      const organization = EmberObject.create({ canCollectProfiles: true });
```

Comment:
Supprimer ce parametre dans l'organisation
```suggestion
      const organization = EmberObject.create();
```

Reply:
Si je le supprime mon test fail, le canCollectProfile dans le hbs permet d'afficher les radio button, 
Evaluer les participant ainsi que collecter les profils.

Reply:
et `canCollectProfiles: false` tu as essayé ?

Reply:
Yes, si c'est false, je ne passe pas dans ce if.
<img width="952" alt="Capture d’écran 2020-09-09 à 09 40 11" src="https://user-images.githubusercontent.com/13931126/92569115-7885ed00-f280-11ea-80ee-798b8116e82e.png">

Peut être revoir la condition ?

Reply:
Par conséquence je n'atteins pas le bouton Evaluer les participant et n'est donc pas accès au lien de la documentation.